Plugins/Public/getrep/Main.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __MAIN_H__
```

Comment:
I wonder if we can leave this file out - conn_plugin and hookext_plugin don't have it